src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -303,32 +307,57 @@ export class GcpCreateServerApp extends LitElement {
     } else {
       if (!this.billingAccounts || this.billingAccounts.length === 0) {
         this.showBillingAccountSetup();
+        // Check every five seconds to see if an account has been added.
+        this.billingAccountsRefreshLoop = window.setInterval(() => {
+          this.refreshBillingAccounts();
+        }, 5000);
       } else {
         this.showProjectSetup(this.project);
       }
     }
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.stopRefreshingBillingAccounts();
+  }
+
   private init() {
     this.currentPage = '';
     this.selectedProjectId = '';
     this.selectedBillingAccountId = '';
+    this.stopRefreshingBillingAccounts();
   }
 
   private showBillingAccountSetup(): void {
     this.currentPage = 'billingAccountSetup';
   }
 
-  private async handleBillingVerificationNextTap(): Promise<void> {
-    this.showProjectSetup();
+  private async refreshBillingAccounts(): Promise<void> {
+    this.billingAccounts = await this.account.listBillingAccounts();
+    // FIXME: listBillingAccounts() can reject, resulting in an uncaught
```

Comment:
If we want to put in some UI for now we can send an event to be handled by appRoot and use `showError`, something like https://github.com/Jigsaw-Code/outline-server/blob/master/src/server_manager/web_app/app.ts#L272.

Reply:
I adjusted this comment.  I think a toast is a reasonable possibility, but I'd prefer to land this PR and discuss the error handling question separately.